app/model/Payment/Payment/EmailRecipient.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Payment\Payment;
+
+use Doctrine\ORM\Mapping as ORM;
+use Model\Payment\Payment;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="pa_payment_email_recipients")
+ */
+class EmailRecipient
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer", options={"unsigned"=true})
+     */
+    private int $id;
+
+    /** @ORM\ManyToOne(targetEntity=Payment::class, inversedBy="emailRecipients") */
+    private Payment $payment;
+
+    /** @ORM\Column(type="text") */
+    private string $email_address;
```

Comment:
```suggestion
    private string $emailAddress;
```

\+ prosím ten custom type

Reply:
ok